frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -140,28 +140,26 @@ class TableItem extends React.Component<{
                     onMouseLeave={(e) => this.setState({ hovered: false })}
                     component='th'
                     key={index}
+                    classes={{
+                        root: classNames({
+                            [classes.tableHeader]: true,
+                        }),
+                    }}
                     style={
                         index === 0
                             ? {
-                                  paddingLeft: editPermission ? '30px' : '70px',
-                                  position: 'sticky',
-                                  top: 0,
-                                  backgroundColor: '#F8F9FA',
-                                  zIndex: 1,
-                              }
-                            : {
-                                  position: 'sticky',
-                                  top: 0,
-                                  backgroundColor: '#F8F9FA',
-                                  zIndex: 1,
+                                  paddingLeft: editPermission ? '30px' : '40px',
```

Comment:
We can also make this a class and do something like [classes.firstColumnHeader]: index === 0 and not use style here at all